guides/common/modules/con_limitations-of-installation-in-an-ipv6-network.adoc

```diff
@@ -0,0 +1,7 @@
+[id="limitations-of-installation-in-an-ipv6-network_{context}"]
+= Limitations of {Project} Installation in an IPv6 Network 
+
+{Project} installation in an IPv6 network has the following limitations:
+
+* DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore {SmartProxy} DHCP plug-in that provides DHCP management is limited to IPv4 subnets. In this scenario, DHCP IPv6 must be deployed as a separate unmanaged service to bootstrap client nodes into grub2, which then configures IPv6 networking either using DHCPv6 or via static IPv6 address.
```

Comment:
that provides management of ISC DHCP

is probably better because we also ship Infoblox DHCP and that one could work (we haven't tested actually).

Reply:
Where shall I put your suggestion @lzap ?